File: Beecrowd/2497 Contando Ciclos/main.c
#include <stdio.h>

int main()
{
    int n, cycle, i = 1;

    while (1)
    {
        scanf("%d", &n);
        if (n == -1)
            break;
        cycle = n / 2;
        printf("Experiment %d: %d full cycle(s)\n", i++, cycle);
    }

    return 0;
}

File: Beecrowd/2497 Contando Ciclos/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n, cycle, i = 1;

        while (true) {
            n = sc.nextInt();

            if (n == -1)
                break;

            cycle = n / 2;
            System.out.println("Experiment " + (i++) + ": " + cycle + " full cycle(s)");
        }

        sc.close();
    }

}